app/controllers/finance/bank_accounts_controller.rb

```diff
@@ -5,4 +5,18 @@ def index
     redirect_to finance_bank_account_transactions_url(@bank_accounts.first) if @bank_accounts.count == 1
   end
 
+  def import
+    @bank_account = BankAccount.find(params[:id])
+    import_method = @bank_account.find_import_method
+    if import_method
+      count = import_method.call(@bank_account)
```

Comment:
As it's used now, `@backaccount.import` would make more sense.  But perhaps you'd like to go somewhere where this makes sense. Let's see where it goes.

Reply:
since the import methods will be implemented in a plugin i need a place where i can hook them in and it's an clean way for the CSV import in the future